json/Makefile

```diff
@@ -35,4 +35,4 @@ endif
 endif
 
 clean:
-	rm -fr syntax.json 
+	rm -fr syntax.json action_list
```

Comment:
@gtribello is this change related to this PR?

Reply:
Not necessarily.  I think it fixes a typo I made.  When you run that json makefile it creates two files: syntax.json and action list.  If you run make clean those two files should be deleted as these are what the makefile creates.

What should I do to resolve this type of problem in general.  Just commit and push it separately and say it fixes a typo?

Reply:
It's fine I was just wondering.